Compiler/FrontEnd/Inst.mo

```diff
@@ -799,13 +799,13 @@ algorithm
         InnerOuter.INST_INNER(innerElement = SOME(c)) =
           InnerOuter.lookupInnerVar(inCache, env, inIH, inPrefix, n, io);
 
-        (cache,env,ih,store,dae,csets,ci_state,tys,bc,oDA,equalityConstraint,graph) =
+        (cache,env,ih,store,ci_state,graph,csets,dae,tys,bc,oDA,equalityConstraint) =
```

Comment:
No, I changed the order of the function parameters in instClassIn2 (because I used input output in some cases), so that is correct. Otherwise you'd get compilation errors, which is not the issue. The reason this fails is because the bootstrapping becomes too slow, probably because the instantiation cache is messed up somehow.